libcodechecker/server/permissions.py

```diff
@@ -597,7 +595,6 @@ def require_manager(permission, extra_params, user):
 
     return False
 
-# ---------------------------------------------------------------------------
```

Comment:
I would suggest leaving this line in the code. It is better if the "work logic definition" part of this file (permissions classes and methods) and the "permission declaration" part is separated.